control/control.openSUSE.xml

```diff
@@ -288,7 +288,7 @@ textdomain="control"
         <try_separate_home config:type="boolean">true</try_separate_home>
         <limit_try_home>7GB</limit_try_home>
         <root_space_percent>40</root_space_percent>
-        <root_base_size>5GB</root_base_size>
+        <root_base_size>3GB</root_base_size>
```

Comment:
usually other changes in storage-ng branches are properly tagged that it contain something that should be changed later. see https://github.com/yast/yast-storage-ng/blob/master/doc/installer-hacks.md